features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/modules/api/add.jag

```diff
@@ -259,6 +259,29 @@ var getTiers = function () {
     }
 };
 
+var getResourceTiers = function () {
+    var tier,tiers,list=[],log = new Log(),
+            provider = jagg.module("manager").getAPIProviderObj();
```

Comment:
already fixed in commit https://github.com/wso2/carbon-apimgt/commit/f7c4cb6913c12e654009faf2a6631ebf2a9cefc4